modules/core/resources/META-INF/services/software.amazon.smithy.model.traits.TraitService

```diff
@@ -4,6 +4,7 @@ alloy.common.HexColorCodeFormatTrait$Provider
 alloy.common.LanguageCodeFormatTrait$Provider
 alloy.common.LanguageTagFormatTrait$Provider
 alloy.DateFormatTrait$Provider
+alloy.DataExamplesTrait$Provider
```

Comment:
nit pick: but can you keep lexicographical ordering?

Reply:
Damn I even tried to.. haha I don't know the alphabet 😆